worker/Dockerfile

```
@@ -8,7 +8,7 @@ RUN GO111MODULE=on go mod download
 ARG version=dev
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags "-w -extldflags '-static' -X main.VersionTag=${version}" -o /worker cmd/worker/worker.go
 
-FROM alpine:3.16
+FROM alpine:3.15
```

```suggestion
FROM alpine:3.16
```
I think this can reverted since the issues with 3.16 should be resolved by now.